dockerfiles/framework/minio-nats/README.md

```diff
@@ -0,0 +1,32 @@
+[![build status](https://gitlab.balldayton.com/ball-docker/minio-nats/badges/master/build.svg)](https://gitlab.balldayton.com/ball-docker/minio-nats/commits/master)
+
+Minio and NATS
+==============
+An image containing minio and NATS streaming server setup as an S3/SQS analog.
+
+Running this image will server SQS compatible messages via NATS streaming server on port 4222 and minio on port 9000.
+This setup is quite useful for developing S3 enabled applications.
+Also included is a platform independent (pure python) wheel for the pynats library which can be used to subscribe to the NATS events.
+It should work on python 2 or python 3. [https://github.com/mcuadros/pynats]
```

Comment:
I'm strongly averse to including external code in our repo, especially of the pre-release variety (0.0.1) and binary form (whl). I'm aware its just a zip and has pure python in there, but propose we hold off until Python 3 and asyncio-nats.